lib/page/settings.dart

```diff
@@ -29,6 +29,7 @@ class _SettingsPageState extends State<SettingsPage> {
       'theme': 'light',
       'search_content': true,
       'editor_mode_switcher': true,
+      'editor_pair_brackets': true,
```

Comment:
Thank you for your contribution! Could you please change the default value of the `editor_pair_brackets` preference to `false`?

Reply:
Of course, just pushed!